package/crds/categories.magento.crossplane.io_categories.yaml

```diff
@@ -4,18 +4,18 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.12.1
-  name: mytypes.sample.template.crossplane.io
+  name: categories.categories.magento.crossplane.io
```

Comment:
@ca7alindev Do not give me auto replaced version, check what you doing. categories, categories, categories, how much categories shall be in name. Please keep developer mind.